doc/README_wcoss_dell_p3_intel.txt

```diff
@@ -0,0 +1,98 @@
+Setup instructions for NOAA WCOSS DELL machine using Intel-18.0.1.163
+
+NOTE: set "export INSTALL_PREFIX=..." as required for your installation
+
+export INSTALL_PREFIX=/gpfs/dell2/emc/_YOUR_GROUP_/noscrub/$USER/wrk/UFS_build
```

Comment:
This should be a shared location, since wcoss_dell_p3 is a preconfigured platform.